.github/workflows/regression-report.yml

```diff
@@ -222,24 +223,42 @@ jobs:
 
       - name: Find input file
         id: find_input_file
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_NUMBER: ${{ github.event.issue.number }}
         run: |
           INPUT_FILE="${{ needs.parse_comment.outputs.input_file }}"
-          WORKSPACE="${{ github.workspace }}"
+          # Fetch the PR details
+          PR_DETAILS="$(curl --fail-with-body -s -H "Authorization: token $GITHUB_TOKEN" \
+            "https://api.github.com/repos/${{ github.repository }}/pulls/$PR_NUMBER")"
```

Comment:
We don't need this....
We just need to make `Restore PR Checkout` to work.

Reply:
I tried a lot many ways, but i can't get the forked branch.

Reply:
Let's clone it from scratch.

Reply:
@romani please review